stylesheets/livingstyleguide/_color-swatches.scss

```diff
@@ -26,12 +26,38 @@
     z-index: -1;
   }
 
+  &:hover:before {
+    color: black;
+    cursor: pointer;
+  }
+
   &:after {
+    cursor: pointer;
     display: block;
     font-weight: normal;
     text-align: center;
   }
 }
+.-lsg-copy-color{
+  background-color: hsla(333, 100%, 0%, 0.6);
+  border-radius:7%;
```

Comment:
Please don’t hard-code the border-radius here. You can make use of the variable defined here:

https://github.com/livingstyleguide/livingstyleguide/blob/v2/stylesheets/_livingstyleguide.scss#L30